README.md

```diff
@@ -6,8 +6,8 @@ Swanky node is a Substrate based blockchain configured to enable `pallet-contrac
 - [pallet-contracts](https://github.com/paritytech/substrate/tree/master/frame/contracts) (polkadot-0.9.37).
```

Comment:
In `astar-collator` we use custom `pallet-contracts` so we should use that in swanky too.
https://github.com/AstarNetwork/Astar/blob/8e49d79e653af3c9946f9d17c1ca72973bf1443e/Cargo.toml#L19-L20